src/game.py

```diff
@@ -0,0 +1,61 @@
+import random
+from dataclasses import dataclass
+
+
+@dataclass
+class Game:
+    """Class for overall game."""
+
+    # intializing objects for class
+    characters_playing: list[dict[str, str]]
+    spells: list[dict[str, str]]
+
+    def bracket_maker(self) -> list[dict[str, str]]:
+        """Creates bracket uses selected players"""
+        character_pairs = list(
+            zip(self.characters_playing[::2], self.characters_playing[1::2])
+        )
+        return character_pairs
+
+    def print_duel_pairs(self, character_pairs_list: list[dict[str, str]]) -> None:
+        """Prints duel games before being played."""
+        for duel_pairs in character_pairs_list:
+            print("Duel: ", duel_pairs[0]["Name"], "vs.", duel_pairs[1]["Name"])
+            print("   ")
+
+
+# intitializing character list
+def read_data(file_name: str) -> list[dict[str, str]]:
```

Comment:
this function reminds me of parse_data lol I think it looks good.

Reply:
Yes, it ultimately does the same thing.